doc/devices/remote.rst

```diff
@@ -0,0 +1,79 @@
+The Remote device
+=================
+
+Pennylane's Remote device gives access to remote backends from Strawberry Fields including
+`hardware backends <https://strawberryfields.ai/photonics/hardware/index.html>`_.
+
+Accounts and Tokens
+~~~~~~~~~~~~~~~~~~~
+
+By default, the ``strawberryfields.remote`` device will attempt to use an
+already active or stored Strawberry Fields account. If the device finds no
+account it will raise a warning:
+
+.. code::
+
+    'WARNING:strawberryfields.configuration:No Strawberry Fields configuration file found.'
+
+You can use the ``strawberryfields.store_account("<my_token>")`` function to
+permanently store an account.  Alternatively, you can use the `Strawberry
+Fields command line interface for configuration
+<https://strawberryfields.readthedocs.io/en/stable/code/sf_cli.html>`__.
+
+.. warning:: Never publish code containing your token online.
+
+Usage
+~~~~~
+
+You can instantiate the Remote device in PennyLane as follows:
+
+.. code-block:: python
+
+    import pennylane as qml
+
+    dev = qml.device('strawberryfields.remote', backend="X8", shots=10, sf_token="XXX")
+
+The device can then be used to create supported circuits to define and evaluate
+QNodes within PennyLane. Refer to the `Strawberry Fields hardware page
+<https://strawberryfields.readthedocs.io/en/stable/introduction/photonic_hardware.html>`__
+for more details on circuit structures, backends to use and getting an
+authentication token.
+
+As an example, the following simple example defines a :code:`quantum_function`
+circuit that first applies two-mode squeezing on the the vacuum state, followed
+by beamsplitters, and then returns the photon number expectation. This function
+is then converted into a QNode which is placed on the
+:code:`strawberryfields.ai` device:
+
+.. code-block:: python
+
+    @qml.qnode(dev)
+    def quantum_function(theta, x):
+        qml.TwoModeSqueezing(1.0, 0.0, wires=[0,4])
+        qml.TwoModeSqueezing(1.0, 0.0, , wires=[1,5])
```

Comment:
Sorry, added an extra comma in the prior suggestion. :disappointed: 
```suggestion
        qml.TwoModeSqueezing(1.0, 0.0, wires=[1,5])
```

Reply:
no problem, thank you for spotting it!